### PR TITLE
#164047082 users like or dislike comment

### PR DESCRIPTION
**Pull request by mnswaleh**
#### Description
Enable users to like and dislike comments
- a user like and unlike comments
- user dislike and undo dislike on comments

#### Type of change
New endpoints
```
- POST articles/like_unlike_comment/<pk>
- POST articles/dislike_comment/<pk>
```

#### How Has This Been Tested?
access above endpoint

#### Checklist:
- a user can like comments
- a user can unlike comments
- a user can dislike comments
- a user can undo dislike comment

#### Pivotal Tracker
[finishes #164047082](https://www.pivotaltracker.com/story/show/164047082)

### CODE REVIEW

File: authors/apps/article/urls.py [hunk 1]
```diff
@@ -13,5 +13,8 @@
     path('articles/<slug>/dislike', DislikeAPIView.as_view()),
     path('articles/<slug>/comments/', CommentsAPIView.as_view()),
     path('articles/<slug>/comments/<pk>', RetrieveCommentsAPIView.as_view()),
-    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view())
+    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view()),
+    path('articles/<slug>/like_unlike_comment/<pk>', LikeUnlikeAPIView.as_view()),
```
Comment (JoanNgatia) [inline authors/apps/article/urls.py]: Hey @mnswaleh we can leave this as `articles/ <slug> /like_comment/<pk>'

**Review by JoanNgatia — CHANGES_REQUESTED** [review]

File: authors/apps/article/urls.py (hunk 1 above)
Comment (mnswaleh) [reply authors/apps/article/urls.py]: Ok

File: authors/apps/article/renderer.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import json
 from rest_framework.renderers import JSONRenderer
+from .models import LikeComment
```
Comment (JoanNgatia) [inline authors/apps/article/renderer.py]: @mnswaleh You've imported the model but haven't used it in this file. Please remove it.
Also, revert the changes to spacing on this file.

File: authors/apps/article/urls.py [hunk 3]
```diff
@@ -13,5 +13,7 @@
     path('articles/<slug>/dislike', DislikeAPIView.as_view()),
     path('articles/<slug>/comments/', CommentsAPIView.as_view()),
     path('articles/<slug>/comments/<pk>', RetrieveCommentsAPIView.as_view()),
-    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view())
+    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view()),
+    path('articles/like_unlike_comment/<pk>', LikeUnlikeAPIView.as_view()),
```
Comment (JoanNgatia) [inline authors/apps/article/urls.py]: I also think we should rename this link to just like_comment/pk. the APIView name can remain as is to give more description though

File: authors/apps/profiles/tests/test_api.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 # tests for Endpoints
 
 
-class UserTest(APITestCase):
+class ProfilTest(APITestCase):
```
Comment (JoanNgatia) [inline authors/apps/profiles/tests/test_api.py]: Check on the spelling of this class name.

**Review by JoanNgatia — CHANGES_REQUESTED** [review]

**Review by JoanNgatia — APPROVED** [review]
